Explorer/Assets/Scripts/Global/Dynamic/DynamicGlobalRealmNavigator.cs

```diff
@@ -32,13 +31,13 @@
 
 namespace Global.Dynamic
 {
-    public class RealmNavigator : IRealmNavigator
+    public class DynamicGlobalRealmNavigator : IRealmNavigator
     {
         private readonly URLDomain genesisDomain = URLDomain.FromString(IRealmNavigator.GENESIS_URL);
 
         private readonly ILoadingScreen loadingScreen;
         private readonly IMapRenderer mapRenderer;
-        private readonly IRealmController realmController;
+        private readonly IGlobalRealmController realmController;
```

Comment:
Same here, we dont have a non global one. Why the name change?

Reply:
Because it contains a `GlobalWorld`, something that is only required in this assembly (DCL.Plugins), so we keep it separated from `IRealmController` thus can make a proper dependency management.